### PR TITLE
Allow version spec in `--ignore-packages`

### LINKED ISSUE #151: Allow a version to be specified in `--ignore-packages`
:wave: 

It'd be great if I could specify a version along with the name in `--ignore-packages`. 

I can hand-curate this list for my org based on the current version (oh no `foo:0.2.1` has `UNKNOWN`, but the `LICENSE.txt` is Apache, so we're OK). But if someone updated the version it a newer version with a different license we shouldn't ignore it in case the license changes.

### PR DESCRIPTION
**Pull request by thejcannon**
Fixes #151

### CODE REVIEW

**Comment by raimon49** [comment]
This proposal looks good 👍 

Can you please put in the README the specs for when you want to ignore `pkg_name:version`?

**Comment by thejcannon** [comment]
Done! Looks like CI had a code coverage issue on the last commit. Let's see if this one makes it happy.

File: README.md [hunk 1]
```diff
@@ -351,6 +350,16 @@ Package names of arguments can be separated by spaces.
...
+Packages can also be specified with a version, only ignoring that specific version.
+
+```bash
+(venv) $ pip-licenses --with-system --ignore-packages django pytz:2017.3
+ Name        Version  License
+ prettytable 3.5.0    BSD License
+ setuptools  38.5.0   UNKNOWN
+ wcwidth     0.2.5    MIT License
+```
+
```
Comment (raimon49) [inline README.md:363]: Thanks, looks good.

Do you use any automated formatting to edit your Markdown files? The diff is larger. If possible I would be happy if you could add just this section.

Comment (thejcannon) [reply README.md:363]: Oh yeah whoops, `prettier` is enabled by default in my editor. let me revert those and save without formatting.

Comment (raimon49) [reply README.md:363]: Thanks for the re-commit and force-push. Include this PR in the next release.

**Comment by raimon49** [comment]
@thejcannon Thanks for your contribution.

The next release candidate version has been uploaded to PyPI.
https://pypi.org/project/pip-licenses/4.2.0rc1/

Install it in your environment and make sure it is working as expected.
```bash
pip install pip-licenses==4.2.0rc1
```

If you have code to fix, please send a PR to the `release-4.2.0` branch.

**Comment by thejcannon** [comment]
I'm already happily using it via a VCS requirements string from my fork 🙃

**Pull request merged**